### PR TITLE
[BZ 1467786] Fix for OPENSHIFT_DEFAULT_REGISTRY setting in env file.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

File: roles/openshift_master/tasks/main.yml [hunk 1]
```diff
@@ -128,8 +128,17 @@
   when: openshift.master.request_header_ca is defined and item.kind == 'RequestHeaderIdentityProvider' and item.clientCA | default('') != ''
   with_items: "{{ openshift.master.identity_providers }}"
 
-- set_fact:
-    openshift_push_via_dns: "{{ openshift_use_dnsmasq | default(true) and openshift.common.version_gte_3_6 and r_openshift_master_clean_install }}"
+- block:
+    # This is an ugly hack to verify settings are in a file without modifying them with lineinfile.
+    # The template file will stomp any other settings made.
+  - name: check whether our docker-registry setting exists in the env file
+    command: "awk '/^OPENSHIFT_DEFAULT_REGISTRY=\'docker-registry.default.svc:5000\'/' /etc/sysconfig/{{ openshift.common.service_type }}-master"
```
Comment (sdodson) [inline roles/openshift_master/tasks/main.yml]: Is this expecting single quotes around the value? We don't include those in the template.

Comment (kwoodson) [inline roles/openshift_master/tasks/main.yml]: I'll remove the quotes and retest.  I did test but must have not hit the following if statement correctly.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Pull request merged**